Tom-tom7m/Homework-1/BONUS-random-address-generator/app.js

```diff
@@ -0,0 +1,29 @@
+/* Optional bonus work on conditionals and functions:
+
+    Write a script that can generate random addresses As a first step, create
+    arrays that contain dummy data for each of the following: street number,
+    street name, city name, state name, and zip code Your script should
+    randomly select one item from each of these arrays and then use them to
+    construct a random address Each time you load or reload the page in a
+    browser, your program should print a new randomly-generated address to the
+    terminal. For example:
+    
+    34578 Dolphin Street, Wonka NY, 44506 */
+
+"use strict";
+
+function getRandomInteger(min, max) {
```

Comment:
nice use of separating out a helper function.

I would also suggest doing something similar to handle finding the integer of the index to use on the array below. This helps a lot with readability and will help keep the code as clean as possible